.github/workflows/doc-workflow.yaml

```diff
@@ -46,7 +46,8 @@ jobs:
         working-directory: ./astarte-device-sdk-python/docs
         run: |
           python -m pip install --upgrade pip
-          pip install setuptools wheel twine requests paho-mqtt cryptography bson PyJWT "Sphinx<7" m2r2 sphinx_rtd_theme
+          pip install "Sphinx<7" m2r2 sphinx_rtd_theme
```

Comment:
I would place those three packages in a `requirements.txt` in the docs folder. Just to make it cleaner and also being easier to run sphinx locally